test/e2e/scenario/rum/sessions.scenario.ts

```diff
@@ -0,0 +1,54 @@
+import { SESSION_COOKIE_NAME } from '@datadog/browser-core'
+import { createTest, flushEvents } from '../../lib/framework'
+import { browserExecute, browserExecuteAsync } from '../../lib/helpers/browser'
+
+describe('rum sessions', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about moving expire and renew session scenario in this suite?